roles/remediate/README.md

```diff
@@ -0,0 +1,80 @@
+# Remediations
+
+**IMPORTANT:** This role is only supported for RHEL 8 systems.
+
+The `remediation` role is to assist in the remediation of a system. This role contains multiple playbooks that can be used to remediate a system for a specific inhibitors that are found during the pre-upgrade analysis.
+
+## Role variables
+
+| Name                    | Default value         | Description                                         |
+|-------------------------|-----------------------|-----------------------------------------------------|
+| leapp_report_location   | /var/log/leapp/leapp-report.json | Location of the leapp report file.       |
+| remediation_playbooks   | see [Remediation playbooks](#remediation-playbooks) | List of available remediation playbooks.|
+| remediation_todo        | []                    | List of remediation playbooks to run.               |
+
+`remediation_todo` is a list of remediation playbooks to run. The list is empty by default. The list can be populated by the titles from [Remediation playbooks](#remediation-playbooks) section. For example:
+
+```yaml
+remediation_todo:
+  - leapp_cifs_detected
+  - leapp_corrupted_grubenv_file
+```
+
+## Remediation playbooks
+
+The list of available remediation playbooks with their corresponding inhibitors titles:
+
+- `leapp_cifs_detected`
+  - **Solves:** Use of CIFS detected. Upgrade can't proceed.  CIFS is currently not supported by the inplace upgrade.
+- `leapp_corrupted_grubenv_file`
+  - **Solves:** Detected a corrupted grubenv file.
+- `leapp_custom_network_scripts_detected`
+  - **Solves:** custom network-scripts detected. RHEL 9 does not support the legacy network-scripts package that was deprecated in RHEL 8.
+- `leapp_deprecated_sshd_directive`
+  - **Solves:** A deprecated directive in the sshd configuration.
+- `leapp_firewalld_allowzonedrifting`:
+  - **Solves:** Firewalld Configuration AllowZoneDrifting Is Unsupported.
+- `leapp_firewalld_unsupported_tftp_client`
+  - **Solves:** Firewalld Service tftp-client Is Unsupported.
+- `leapp_loaded_removed_kernel_drivers`
+  - **Solves:** Leapp detected loaded kernel drivers which have been removed in RHEL 8. Upgrade cannot proceed.
+- `leapp_missing_efibootmgr`
+  - **Solves:** efibootmgr package is required on EFI systems.
+- `leapp_missing_pkg`
+  - **Solves:** Leapp detected missing packages.
+- `leapp_missing_yum_plugins`
+  - **Solves:** Required DNF plugins are not being loaded.
+- `leapp_multiple_kernels`
+  - **Solves:** Multiple kernels installed.
+- `leapp_newest_kernel_not_in_use`
+  - **Solves:** Newest installed kernel not in use.
```

Comment:
I would like to see a warning somewhere to make users aware that the `leapp_multiple_kernels` and `leapp_newest_kernel_not_in_use` playbooks will reboot the host.